static/app/views/performance/queues/queries/useQueuesMetricsQuery.tsx

```diff
@@ -0,0 +1,39 @@
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useSpanMetrics';
+
+type Props = {
+  destination?: string;
+  enabled?: boolean;
+  transaction?: string;
+};
+
+export function useQueuesMetricsQuery({destination, transaction, enabled}: Props) {
+  const mutableSearch = new MutableSearch(
+    'span.op:[queue.task.celery,queue.submit.celery]'
```

Comment:
This is starting to look like some kind of default conditions, right? If yes, worth dumping into `settings.tsx` for the module!

Reply:
good call, updated in `settings.tsx`!